### PR TITLE
chore: remove --container-runtime from AL2 user data

### PR DESCRIPTION
**Pull request by jmdeal**
Fixes #N/A 

**Description**
In light of `--container-runtime` being ignored by the EKS bootstrap script and v1beta1 APIs removing the option to set container runtime, this PR drops the field from the AL2 AMI.

**How was this change tested?**
- `make test`

**Does this change impact docs?**
- [x] Yes, PR includes docs updates 
- [ ] Yes, issue opened: # 
- [ ] No

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: website/content/en/docs/concepts/nodeclasses.md [hunk 1]
```diff
@@ -29,21 +29,21 @@ metadata:
   name: default
```
Comment (jonathan-innis) [inline website/content/en/docs/concepts/nodeclasses.md:29]: These changes look good to me, but looks like you have a bunch of whitespace changes in files where you actually didn't make any edits.

Comment (jmdeal) [reply website/content/en/docs/concepts/nodeclasses.md:29]: Are you referring to the two docs files? Both of these have edits, it's just a single edit to remove the `--container-runtime` flag from the default user data section.

**Review by jonathan-innis — APPROVED** [review]
LGTM 🚀

File: website/content/en/docs/concepts/nodeclasses.md (hunk 1 above)
Comment (jonathan-innis) [reply website/content/en/docs/concepts/nodeclasses.md:29]: Got it. I just did a cursory look over it so I missed that change due to all of the whitespace updates. Not sure if there is a way to reduce the number of whitespace updates that are propagated from your editor. For now, I just enabled the "Hide Whitespace" setting and was able to see it more clearly.

**Pull request merged**